src/legacy/core_plugins/data/public/expressions/expression_renderer.tsx

```diff
@@ -21,27 +21,24 @@ import { useRef, useEffect } from 'react';
 import React from 'react';
 import { Ast } from '@kbn/interpreter/common';
 
-import { ExpressionRunnerOptions, ExpressionRunner } from './expression_runner';
-import { Result } from './expressions_service';
+import { IInterpreterResult } from './lib/_types';
+import { IExpressionLoader } from './lib/loader';
 
 // Accept all options of the runner as props except for the
 // dom element which is provided by the component itself
-export type ExpressionRendererProps = Pick<
```

Comment:
Removing that makes it impossible to pass in things like the context over the react interface. `IExpressionLoaderParams` should be part of the renderer props. They are already passed correctly to the loader: https://github.com/elastic/kibana/pull/42337/files#diff-e2b0614a60b75e9f05d4bec84efb896aR50